### ISSUE TITLE
Links word oddly inside of Facebook Messanger

### ISSUE THREAD

**Issue by bassosimone** [TheTorProject/ooniprobe-ios#128]
This is kinda known, but I feel like it's good anyway to post it here. I wrote a number of run.ooni.io links to a OONI colleague on Facebook Messanger. Upon clicking on the links, the app was not opened, and instead I was directed on run.ooni.io. There, I could click on a button to run the test successfully.

I guess this is okay for a first release, because the functionality is there. What is a bit annoying is that you need to click twice. To confirm you're sure you want to leave Facebook Messenger and to confirm that you want to allow Facebook Messenger to open ooniprobe.

**Comment by lorenzoPrimi** [comment]
@hellais  also here

**Comment by hellais** [comment]
Yes I made some attempts at supporting facebook (not messenger) and even that is a huge pain. The developer docs are conflicting and there seems to be no central place to know what you should do unless you do a bunch of trial and error.

For the moment I suggest we postpone this.

These are some relevant links:
* https://developers.facebook.com/docs/app-ads/deep-linking/
* https://blog.branch.io/how-to-deep-link-on-facebook/

FWIW I did implement what is recommend there, but it doesn't actually work.

**Comment by lorenzoPrimi** [comment]
related to this maybe https://github.com/ooni/probe/issues/814

**Comment by lorenzoPrimi** [comment]
@hellais @bassosimone is this only iOS related or also Android?
